models/yang/sonic/sonic-sflow.yang

```diff
@@ -100,7 +100,6 @@ module sonic-sflow {
                             path "/prt:sonic-port/prt:PORT/prt:PORT_LIST/prt:ifname";
```

Comment:
We need to allow adding lag,vlan and other supported interface types.

Reply:
I just checked with Padman from the team and he said that Phase I of sFlow only supports Physical interfaces, so let's just keep it this way.